docs/code_ref/coordinates.rst

```diff
@@ -130,8 +130,8 @@ coordinates is::
    <SkyCoord (Helioprojective: obstime=2017-07-26T00:00:00.000, rsun=695700.0 km, observer=<HeliographicStonyhurst Coordinate for 'earth'>): (Tx, Ty) in arcsec
        (0., 0.)>
    >>> c.transform_to(frames.HeliographicCarrington)
-   <SkyCoord (HeliographicCarrington: obstime=2017-07-26T00:00:00.000): (lon, lat, radius) in (deg, deg, km)
-      (283.95274241, 5.31701821, 695700.00000125)>
+   <SkyCoord (HeliographicCarrington: obstime=2017-07-26T00:00:00.000): (lon, lat, radius) in (deg, deg, AU)
```

Comment:
This feels like a change we probably shouldn't do in a bugfix? (changing the default unit?)

Reply:
I'm a bit mystified here.  I made this change (and others like it) so that the doc tests would pass, but when I run the same code locally, I still get km, not AU.  I don't know where AU is coming from.

Reply:
Okay, I don't know why I was getting weird answers locally.  (Maybe I was running in an old shell.)  My local results currently match the CI results.

Are you of the mind that I should force everything to "km"?  It wasn't actually a default defined by the frame, but rather the unit was hard-coded as the output of transformations.  It seems a bit unseemly to force the unit if the sole reason is to exactly match the previous output.

Reply:
Ok, I think that makes sense. The idea of units after all is that it doesn't matter what unit it is in, and AU is much nicer than km anyway.